File: GTTS/INL.PersonService.Models/GetPersonsTraining_Item.cs
using System;

namespace INL.PersonService.Models
{
    public class GetPersonsTraining_Item : IGetPersonsTraining_Item
    {
        public long TrainingEventID { get; set; }
        public string Name { get; set; }
        public long PersonID { get; set; }
        public string ParticipantType { get; set; }
        public DateTime? EventStartDate { get; set; }
        public DateTime? EventEndDate { get; set; }
        public string BusinessUnitAcronym { get; set; }
        public string TrainingEventRosterDistinction { get; set; }
        public bool? Certificate { get; set; }
        public string TrainingEventStatus { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/get-persons-training_item.ts


import { IGetPersonsTraining_Item } from './iget-persons-training_item';

export class GetPersonsTraining_Item implements IGetPersonsTraining_Item {
  
	public TrainingEventID: number = 0;
	public Name: string = "";
	public PersonID: number = 0;
	public ParticipantType: string = "";
	public EventStartDate?: Date;
	public EventEndDate?: Date;
	public BusinessUnitAcronym: string = "";
	public TrainingEventRosterDistinction: string = "";
	public Certificate?: boolean;
	public TrainingEventStatus: string = "";
  
}


